client/client.go

```diff
@@ -53,6 +53,17 @@ type ErrExpired struct {
 	signed.ErrExpired
 }
 
+// ErrInvalidRemoteKeyType is returned when the server is requested to manage
+// an unsupported key type
+type ErrInvalidRemoteKeyType struct {
```

Comment:
"key type" is overloaded as it's already used to signify the algorithm for a key. This should probably be `ErrInvalidRemoteRole`